### PR TITLE
Allow developers to update their Dotnet SDK version

### PR DESCRIPTION
**Pull request by Thorium**
Just a minor update to get this easier to compile for new people.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
     "version": "5.0.202"
+    ,"rollForward": "minor"
```
Comment (bartelink) [inline global.json:4]: How did you pick this, would latestMajor not work/be better?
also suggest not using that formatting

Comment (Thorium) [reply global.json:4]: Minor is more safe, major typically needs a separate PR for compatibility (see the .NET 6 PRs #605 and #606). Minor forward is just saying that if you have the latest .NET 5 (v.408) installed, you can still compile this repo that is using v.202.

I added the comma intentionally to this line, to not conflict the .NET 6 PR, which is modifying separately the version line above, as we would like to this rolling-minor to be used still after major version update too.

Comment (bartelink) [reply global.json:4]: Ah, you're way ahead of me; thanks for taking the time to explain in detail!

Comment (Numpsy) [reply global.json:4]: > Minor is more safe, major typically needs a separate PR for compatibility

yup, I hit a few errors trying to build with the 6 or 7 SDKs that need either code fixes or further changes elsewhere

**Pull request merged**